integrations/aws-ecs-monitoring.mdx

```diff
@@ -38,80 +49,296 @@ For Each ECS Task that you want to monitor, add this one sidecar container in yo
 }
 ```
 
-### Metrics Collection
-Adding the sidecar container as mentioned above, will automatically start collecting metrics data from your ECS tasks and containers.
-Middleware Agent using AWS ECS Task Metadata Endpoint to fetch metrics data.
+### Collect Metrics 
 
-### Logs Collection
+Adding the sidecar container mentioned above will automatically collect metrics data from your ECS tasks and containers. The
+Middleware Agent uses the AWS ECS Task Metadata Endpoint to fetch metrics data.
+
+### Collect Logs
 
 We offer a few different methods to forward your ECS task logs to Middleware.
 
-#### Method 1 :  Forward ECS Task logs to Middleware
+### Forward ECS Task logs to Middleware
```

Comment:
Think this should be a sub-category to "Collect Logs". It'll help the organization on the right side bar.